lib/listeners/in.js

```diff
@@ -1,6 +1,13 @@
 const slack = require('@slack/client');
 const users = require('../models/users');
 
+const usePrettyAnnounce = true;
+const showFavoredTeam = true;
```

Comment:
So we can put these behind environment variables, but since there's no tests (I need to work on that) I'm good with less code if you want to get rid of the original format and go with this.